subsys/net/ip/net_context.c

```diff
@@ -2401,9 +2401,21 @@ static int recv_udp(struct net_context *context,
 
```

Comment:
Could you change the `This commit fixes the #70020.` from the commit message into `Fixes #70020` and place it before the signed-off-by line. This way it is easier to grep the log later if one needs to figure out the exact commit that fixed it. See examples in other commits in the tree

Reply:
Okay, I will fix it.